### PR TITLE
docs: fixing ci scripts

### PR DESCRIPTION
**Pull request by callmehiphop**

### CODE REVIEW

**Comment by stephenplusplus** [comment]
Does this fix #1778?

**Comment by callmehiphop** [comment]
@stephenplusplus That's my hope, I put a don't merge on it to try and test it out.

**Comment by callmehiphop** [comment]
@stephenplusplus I added a bogus commit to trigger docs/bigquery tests and it looks ok on AppVeyor - https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-node/build/1.0.534/job/vaneo6aud2uvf49t.

**Comment by stephenplusplus** [comment]
Self merge when ready :)

File: packages/bigquery/src/index.js [hunk 1]
```diff
@@ -71,6 +71,7 @@ function BigQuery(options) {
   };
 
   common.Service.call(this, config, options);
+
```
Comment (stephenplusplus) [inline packages/bigquery/src/index.js]: Soooo bogus.

**Pull request merged**